pkg/limits/frontend/http.go

```diff
@@ -61,3 +102,49 @@ func (f *Frontend) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		RejectedStreams: resp.RejectedStreams,
 	})
 }
+
+// PartitionConsumersCacheHandler handles the GET request to display the cache.
+func (f *Frontend) PartitionConsumersCacheHandler(w http.ResponseWriter, _ *http.Request) {
+	data := struct {
+		Entries map[string][]int32
+	}{
+		Entries: make(map[string][]int32),
+	}
+
+	cache := f.streamUsage.(*RingStreamUsageGatherer).cache
```

Comment:
That's not what I meant 😄 We are still doing an unsafe type conversion, it's just been moved out of a method and into the HTTP handler. Why not store a reference to the cache in the handler instead?